gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -302,10 +309,9 @@ static GncInvoice* invoice_from_split (Split* split);
 #define PASTE_TRANSACTION_TIP            N_("Paste the transaction from the clipboard")
 #define DUPLICATE_TRANSACTION_TIP        N_("Make a copy of the current transaction")
 #define DELETE_TRANSACTION_TIP           N_("Delete the current transaction")
-#define ASSOCIATE_TRANSACTION_TIP        N_("Update Association for the current transaction")
-#define ASSOCIATE_TRANSACTION_OPEN_TIP   N_("Open Association for the current transaction")
-#define ASSOCIATE_TRANSACTION_REMOVE_TIP N_("Remove the association from the current transaction")
-#define JUMP_ASSOCIATED_INVOICE_TIP      N_("Open the associated invoice")
+#define ASSOCIATE_TRANSACTION_TIP        N_("Manage the document associated with the current transaction")
+#define ASSOCIATE_TRANSACTION_OPEN_TIP   N_("Open the associated document for the current transaction")
+#define JUMP_ASSOCIATED_INVOICE_TIP      N_("Jump to the associated invoice")
```

Comment:
In the tip we can be more explicit:
"Jump to the associated bill, invoice or voucher"

Reply:
I don't like "corresponding" at all, but I'm not wild about "associated" either.  

For jumping to the other register it's the same transaction, neither corresponding nor associated. IIUC the operation that connects a transaction to a bill/invoice/voucher is called posting so maybe this tooltip could be "Jump to the posting bill/invoice/voucher".

If it weren't for the possible confusion about URLs vs. local files I'd go for changing "associate" to "link" for transactions and external documents. Attach presents the same confusion potential in the other direction. Possible synonyms connect, couple, join, and tie aren't terribly satisfying either.